config_defaults/subtests/docker_cli/start.ini

```diff
@@ -8,6 +8,8 @@ docker_run_timeout = 30.0
 docker_attach = yes
 #: Run interactive container -i
 docker_interactive = yes
+#: docker command wait time
```

Comment:
Comments are best when they explain / clarify an intention. In this case, what would you think of something like "time, in seconds, to wait for command to complete."